services/src/main/java/org/jboss/windup/web/services/rest/WindupEndpoint.java

```diff
@@ -69,4 +69,8 @@
     @GET
     @Path("coreVersion")
     String getCoreVersion();
+
+    @GET
+    @Path("reloadRules")
+    void reloadRules();
```

Comment:
Hmm, I wonder then if it does the same thing, if it should also go to RuleEndpoint?